fe/fe-core/src/main/java/org/apache/doris/analysis/NullLiteral.java

```diff
@@ -101,6 +101,11 @@ public String getStringValue() {
         return "NULL";
     }
 
+    @Override
+    public String getStringValueForArray() {
+        return "null";
```

Comment:
the null value inside an array is represented as "null", for exampe:
`[null, null]`.
Not same as other primitive type to represent as `\N`.
I will add some comment to this method.